scripts/adjustEnvVars.sh

```diff
@@ -165,14 +168,9 @@ MAKE="$MAKE1 --jobs=$JOBS"
 
 KERNEL_NAME=$(uname -s | tr '[:upper:]' '[:lower:]')
 
-function rmStep(){
-  rm -rf "$@"
-  rmdir -p --ignore-fail-on-non-empty `dirname "$@"`
```

Comment:
The `rmdir -p` was being used to remove empty folders when a step failed. I know the `--ignore-fail-on-non-empty` flag did problems on OSX, but could be done some checking of the platform and disable the flag on OSX? Maybe it could be done using a `$RMDIR` variable on the top of the file...